app/renderer/src/containers/AppContainer.js

```diff
@@ -1,26 +1,30 @@
 import { ipcRenderer } from 'electron';
-import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
 import App from '../components/App';
-import * as actionCreators from '../actions/creators';
 import {
   IPC_WINDOW_RESIZE,
+  IPC_WINDOW_COLLAPSE,
+  IPC_WINDOW_EXPAND,
   IPC_LOAD_THEME,
   IPC_QUERY_COMMAND,
 } from '../../../ipc';
 
-const AppContainer = class extends Component {
+export default class AppContainer extends Component {
   static displayName = 'AppContainer';
 
   state = {
     // the current query value
     query: '',
+    // the current set of results
+    results: [],
     // the current theme
     theme: {},
     // currently pressed keys
     keys: [],
+    // details pane content
+    details: '',
```

Comment:
I don't think comment would be necessary if this was named `detailsPaneContent` already.
WDYT?

Reply:
`detailsPaneContent` has since been removed. We're just using `details` for short really here.

Reply:
Ah I just re-read the comment; makes sense.

Reply:
Let's just leave for now, we can rename props to make more sense in a sep. diff.